src/semeio/fmudesign/fmudesignrunner.py

```diff
@@ -77,24 +77,20 @@ def main() -> None:
         print("Worksheets changed from default:")
         print(sheetnames)
 
-    if isinstance(args.config, str):
-        if not Path(args.config).is_file():
-            raise OSError(f"Input file {args.config} does not exist")
-        input_dict = excel2dict_design(args.config, sheetnames)
+    if not Path(args.config).is_file():
+        raise OSError(f"Input file {args.config} does not exist")
+    input_dict = excel2dict_design(args.config, sheetnames)
 
     if args.config == args.destination:
         raise OSError(
             f'Identical name "{args.config}" have been provided for the input'
             "file and the output file. "
-            " Exiting....."
         )
 
     design = DesignMatrix()
 
     design.generate(input_dict)
 
-    Path(args.destination).parent.mkdir(exist_ok=True, parents=True)
```

Comment:
Is this a behavioural change, or is this removal cancelled by the addition in `to_xls()` above?
I would have like to see this change isolated in its own commit with its own  comment (to ease future debugging).

Reply:
I moved it to `to_xls()`, so no behavioural change.

Reply:
Yes, I would like commits that describe what is being performed from the change, making the history readable. We try to follow https://cbea.ms/git-commit/

Reply:
(make it into a separate commit if it is unrelated to other changes in the code only. But it should be possible to read the commit log and check the diff in the future, and quickly understand which changes belong to which commit comment)